core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/kubernetes/reconfigure-rabbitmq-app.yml

```diff
@@ -45,3 +45,28 @@
       when:
         - not rabbit_mq_namespace.stdout == ""
         - not image_registry_address in rabbit_mq_repository.stdout
+    - name: upgrade-master | Get rabbitmq pod names
+      environment:
+        KUBECONFIG: /home/{{ admin_user.name }}/.kube/config
+      shell: |-
```

Comment:
I see that there is no pipe so pipefail is not needed. I would change `shell` module to `command`.